setup.py

```diff
@@ -1,22 +1,44 @@
-import setuptools
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
 
-setuptools.setup(
-    name="barril",
-    version="0.1",
-    url="http://github.com/ESSS/barril",
-    author='ESSS',
+"""The setup script."""
+
+from setuptools import setup, find_packages
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('HISTORY.rst') as history_file:
+    history = history_file.read()
+
+requirements = [ ]
+
+setup(
+    author="ESSS",
     author_email='foss@esss.co',
-    license = 'MIT',
-    description="Python package to manage units for physical quantities",
-    packages=setuptools.find_packages(),
-    install_requires=[],
     classifiers=[
         'Development Status :: 2 - Pre-Alpha',
-        'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: MIT License',
+        'Natural Language :: English',
         'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
     ],
-)
\ No newline at end of file
+    description="Python package to manage units for physical quantities",
+    extras_require={
+        'docs': [ 'sphinx >= 1.4', 'sphinx_rtd_theme', 'sphinx-autodoc-typehints']
```

Comment:
Create a new "extras" here for testing:

```python
'testing': ['pytest', 'pytest-cov'],
```

And use the [`extras`](https://tox.readthedocs.io/en/latest/config.html#confval-extras=MULTI-LINE-LIST) option in `tox.ini` to install it during testing. 👍